app/js/components/manageApps/AddonList.jsx

```diff
@@ -30,7 +30,15 @@ export const AddonList = ({ handleDownload, appList, openPage, openModal, update
                 />
               </td>
               <td onClick={() => openPage(app.appDetails)}>
-                <div className="addon-name">{app.appDetails && app.appDetails.name}</div>
+                <div className="addon-name">
+                  {
+                    app.appDetails.started === true || app.appDetails.started === undefined ?
```

Comment:
Why does it return `started status` when `app.appDetails.started` evaluates to undefined? What if the contents are undefined owing to a bug and the task has not been started? What happens then?

Reply:
The owa's do not have a started property so when you call app.appDetails.name it returns undefined. The started condition mainly applies to a module as for owa's they are alway running